Sources/CacheAdvance/FileHandleExtensions.swift

```diff
@@ -21,12 +21,57 @@ extension FileHandle {
 
     // MARK: Internal
 
+    /// A method to read data from a file handle that is safe to call in Swift from any operation system version.
+    func readDataUp(toLength length: Int) throws -> Data {
+        if #available(iOS 13.0, tvOS 13.0, watchOS 6.0, OSX 15.0, *) {
+            return try __readDataUp(toLength: length)
+        } else {
+            return try ObjectiveC.unsafe { readData(ofLength: length) }
+        }
+    }
+
+    /// A method to write data to a file handle that is safe to call in Swift from any operation system version.
+    func write(data: Data) throws {
```

Comment:
nit: This can probably be just `write(_ data: Data)`

Reply:
Yes! But this is Apple's API signature and I needed a different signature here.